web/src/lib/i18n/en.json

```diff
@@ -1176,6 +1176,7 @@
   "tag_assets": "Tag assets",
   "tag_created": "Created tag: {tag}",
   "tag_feature_description": "Browsing photos and videos grouped by logical tag topics",
+  "tag_not_found_question": "Cannot find a tag? Create one <link>here</link>",
```

Comment:
I'd suggest changing this link to include more context to the user for where the link goes. For example:

`Cannot find a tag? <link>Create a tag here</link>.`

Screen readers have the ability to give a user a list of all the links on a page, and if the word "here" is the only content in the link it can be hard to know what the link is supposed to do. Including more context in the link like "Create a tag here" is more descriptive and solves this issue.